website_multi_theme/models/website.py

```diff
@@ -85,15 +88,10 @@ def _duplicate_view_for_website(self, pattern, xmlid, override_key):
         result = pattern.copy({
             "active": pattern.was_active,
             "arch_fs": False,
-            "customize_show": False,
```

Comment:
*Copied message to don't miss it:*

@Yajo is there any reason why this was set to False? Do you remember the case when it's required?